### PR TITLE
monaspace: init at v1.000

### PR DESCRIPTION
**Pull request by jcdickinson**
## Description of changes

This adds the "Monaspace type system" to nixpkgs. It is a just-released font from Github. More information: https://monaspace.githubnext.com/

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/monaspace/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  pname = "monaspace";
+  version = "1.000";
+
+  src = fetchzip {
+    url = "https://github.com/githubnext/monaspace/releases/download/v${version}/monaspace-v${version}.zip";
+    hash = "sha256-H8NOS+pVkrY9DofuJhPR2OlzkF4fMdmP2zfDBfrk83A=";
+    stripRoot = false;
+  };
+
+  outputs = [ "out" "otf" "variable" ];
```
Comment (AndersonTorres) [inline pkgs/data/fonts/monaspace/default.nix:13]:
```suggestion
  outputs = [ "out" "variable" ];
```

There is no sense on duplicate the files.

File: pkgs/data/fonts/monaspace/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Monaspace";
+    longDescription = "An innovative superfamily of fonts for code.";
+    homepage = "https://monaspace.githubnext.com/";
+    license = licenses.ofl;
+    platforms = platforms.all;
+    maintainers = [];
```
Comment (AndersonTorres) [inline pkgs/data/fonts/monaspace/default.nix:37]: If you are not interested in keeping this package, I  can adopt it.

```suggestion
    maintainers = with  maintainers; [ AndersonTorres ];
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/data/fonts/monaspace/default.nix (hunk 1 above)
Comment (jcdickinson) [reply pkgs/data/fonts/monaspace/default.nix]: Done

File: pkgs/data/fonts/monaspace/default.nix (hunk 2 above)
Comment (jcdickinson) [reply pkgs/data/fonts/monaspace/default.nix]: And done, thanks for taking ownership.

**Review by erictapen — APPROVED** [review]
LGTM!

**Comment by AndersonTorres** [comment]
TKS!

**Pull request closed without merging**